main.js

```diff
@@ -1,56 +1,302 @@
-function check() {
-    document.getElementById('f1').innerHTML = document.userform.firstname.value;
-    document.getElementById('f2').innerHTML = document.userform.lastname.value;
-    document.getElementById('f3').innerHTML = document.userform.address.value;
-    document.getElementById('f4').innerHTML = document.userform.city.value;
-    document.getElementById('f5').innerHTML = document.userform.state.value;
-    document.getElementById('f6').innerHTML = document.userform.zipcode.value;
-    document.getElementById('f7').innerHTML = document.userform.phone.value;
-    document.getElementById('f8').innerHTML = document.userform.EMail.value;
+"use strict"
+function check(event) {
+    
+    event.preventDefault();
+    
+    var firstNameIsValid = validateFirstName();
+    if ( !firstNameIsValid) { 
 
+        return false;
+    }
+
+    var lastNameIsValid = validateLastName();
+    if ( !lastNameIsValid) { 
 
+        return false;
+    }
+
+    var addressIsValid = validateAddress();
+
+    if(!addressIsValid) {
+
+        return false;
 }
 
 
+var cityIsValid = validateCity();
+if (!cityIsValid) {
 
-//         function allLetter(firstname)
-//   {
-//    var letters = /^[A-Za-z]+$/;
-//    if(firstname.value.match(letters))
-//      {
-//       return true;
-//      }
-//    else
-//      {
-//      alert("message");
-//      return false;
-//      }
-//   }
+    return false;
+}
 
+    
+var stateIsValid = validateState();
+    if (!stateIsValid)  {
 
-var verifyPswd = function () {
-    if (document.getElementById('password').value ==
-        document.getElementById('confirm_password').value) {
-        document.getElementById('message').style.color = 'green';
-        document.getElementById('message').innerHTML = 'MATCHING';
-    } else {
-        document.getElementById('message').style.color = 'red';
-        document.getElementById('message').innerHTML = 'NOT MATCHING';
+        return false;
+    }
+
+    var zipcodeIsValid = validateZipcode();
+    if (!zipcodeIsValid) {
+
+        return false;
+    }
+
+    var emailIsValid = validateEmail();
+    if (!emailIsValid) {
+
+        return false;
+    }
+
+    var phoneIsValid = validatePhoneNumber();
+    if (!phoneIsValid) {
+
+        return false;
     }
+
+
+    var passwordIsValid = verifyPswd();
+    if (!passwordIsValid) {
+
+        return false;
+
+    }
+
+  
+
+
+
+    var userFirstName = document.createElement('div');
+                        userFirstName.textContent = document.userform.firstname.value;
+
+
+    var printParent = document.getElementById('print');
+                        printParent.appendChild(userFirstName);
+
+    var userLastName = document.createElement('div');
+                        userLastName.textContent = document.userform.lastname.value;
+                        document.getElementById('print');
```

Comment:
this line isn't doing anything. you've already got `printParent` and are just appending to that every time, which is totally fine, so you can remove this and all following instance of this

Reply:
i understand the error. deleted line.